icu4c/source/tools/genccode/genccode.c

```diff
@@ -84,6 +85,7 @@ static UOption options[]={
 #ifdef CAN_GENERATE_OBJECTS
 /*5*/UOPTION_DEF("object", 'o', UOPT_NO_ARG),
```

Comment:
>/*5*/ [](start = 0, length = 5)

unrelated/aside, but I don't think this is number 5...

Reply:
Also noticed that.

Reply:
Updated.